Procfile

```diff
@@ -0,0 +1 @@
+web: node app.js
```

Comment:
can you add the Procfile to gitingore or does it make sense to commit it? It's for heroku, right?

Reply:
Added it to ignore for now. Yes, for Heroku.